test/helper/index.js

```diff
@@ -226,7 +226,7 @@ export function inject(fn) {
       );
     }
 
-    BPMN_JS.invoke(fn);
+    return BPMN_JS.invoke(fn);
```

Comment:
What would be great is an actual test case that verifies this capability within bpmn-js.

We could also think about adding the same thing (in terms of infrastructure) to diagram-js.